.prettierignore

```diff
@@ -34,4 +34,8 @@ cypress/videos
 cypress/downloads
 
 # fleetdm.com website (uses its own formatting conventions)
-website/
\ No newline at end of file
+website/
+
+# certain JS files that are not meant to be formatted
+frontend/components/FleetAce/mode.ts
+frontend/components/FleetAce/theme.ts
```

Comment:
I'm curious why this was needed as part of this PR since these are old files that don't appear to have changed.

Reply:
That's what I was wondering but, for some reason, the CI process failed 3 times for Prettier checks because of these files.

Reply:
Weird. Maybe a config change made by someone else recently triggered these files? No problem was just curious.